### DISCUSSION TITLE
Option to start without audio

### DISCUSSION THREAD

**Discussion by saatchinorway** [ruffle-rs/ruffle#2998] [Ideas]
I'm helping an artist get his old Flash files available again. Everything works fine with Ruffle, the Flash files are on many separate web pages linked to each other, so I enabled autostart and hidden overlay.

However, the user will need to click once to remove the overlay before any interaction is allowed. The Flash files have no sound, so there is really no need for an overlay to allow sound. Is there an option to play back Flash files without sound, so the autoplay can work seamlessly?

**Comment by ousia** [comment]
@saatchinorway, I’m not sure I’m getting your point.

There was a similar issue (#1755 and probably other ones).

Autoplay is the default now. But Chrome seems not to allow it.

BTW, I just found https://github.com/ruffle-rs/ruffle/issues/2746#issuecomment-765381494, which might address your question.

I never use Chrome, but it seems to work (even with files containing sound).

I hope it helps.

**Comment by Herschel** [comment]
If you are using the polyfill, you can add this script tag before your `ruffle.js` script tag:
```
<script>
    window.RufflePlayer = {
      config: {
        autoplay: "on",
        unmuteOverlay: "hidden",
      }
    };
</script>
```
This will enable autoplay and disable the "click to unmute" overlay.

**Reply by saatchinorway** [reply]
Yes, thanks, this is exactly the config I have, I have enabled autoplay and hidden the unmute overlay. 

But, although the overlay is hidden, the user still has to click an extra time on the flash to interact with it. 

An example: The flash loads, animation starts and everything looks great. The user tries to click a menu item in the flash file, but has to click twice; once to dismiss the (hidden) overlay and then once more to actually activate the menu item.

This is both in Chrome and Safari.

**Reply by Herschel** [reply]
Oops, I'd consider that a bug. Could you file an issue on the repo? Thank you!

**Reply by n0samu** [reply]
For the record, this seems to be fixed now
